spec/Section 7 -- Response.md

```diff
@@ -23,6 +23,24 @@ with key `data`. The value of this entry is described in the "Data" section. If
 the operation failed before execution, due to a syntax error, missing
 information, or validation error, this entry must not be present.
 
+When the response of the GraphQL operation is an event stream, the first value
+will be the initial response. All subsequent values may contain `label` and 
+`path` entries. These two entries are used by clients to identify the the 
+`@defer` or `@stream` directive from the GraphQL operation that triggered this
+value to be returned by the event stream. The combination of these two entries
+must be unique across all values returned by the event stream. 
```

Comment:
🤔 if a `@stream` results in multiple subsequent values in the event stream, won't those all have the same `label` and `path`? Or is `path` expected to include the index into the final array in that case (in which case you can't `@stream` a response in batches and you need to push each element of the array separately which seems very inefficient?)

Reply:
Each payload resulting from a `@stream` corresponds to one list item, so the `path` will contain the index of that element. We discussed different options for custom batching, but there are many different approaches and we ultimately decided not to include it in this proposal. I think specifying batching is something that could be explored separately after there is more widespread usage of `@stream`?